### PR TITLE
Issue #3888: update google java style coverage to 20170228 state

### PR DESCRIPTION
**Pull request by romani**
Issue #3888

### CODE REVIEW

File: src/xdocs/config_javadoc.xml [hunk 1]
```diff
@@ -24,7 +24,9 @@
     <section name="AtclauseOrder">
       <subsection name="Description">
         <p>
-          Checks the order of at-clauses.
+          Checks the order of <a href="http://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDBEFIF">jaavdoc block-tags or javadoc tags</a>.
```
Comment (rnveach) [inline src/xdocs/config_javadoc.xml]: `jaavdoc` -> `javadoc`

File: src/xdocs/google_style.xml [hunk 2]
```diff
@@ -19,7 +19,7 @@
                     This coverage report was created for
                     <a
                         href="http://google.github.io/styleguide/javaguide.html">Google Java Style</a>
-                        (<a href="http://checkstyle.sourceforge.net/reports/google-java-style-20161017.html">cached page</a>)
+                        (<a href="http://checkstyle.sourceforge.net/reports/google-java-style-20170228.html">cached page</a>)
                     , version of 17 November 2016
```
Comment (rnveach) [inline src/xdocs/google_style.xml]: date was not updated

Comment (romani) [inline src/xdocs/google_style.xml]: done

File: src/xdocs/config_javadoc.xml (hunk 1 above)
Comment (romani) [inline src/xdocs/config_javadoc.xml]: done

**Comment by romani** [comment]
I can not rebase and squash as rewriting of history is forbidden in main repo.
@rnveach , as you approve changes, I will squash and rebase manually.

**Comment by romani** [comment]
good catch.

command:
```
$ grep -R "reports/google-java-style\." --exclude="releasenotes.xml" .
./index.xml.vm:        <a href="http://checkstyle.sourceforge.net/reports/google-java-style.html">Google Java Style</a>.
./config_misc.xml:          "http://checkstyle.sourceforge.net/reports/google-java-style.html#s4.8.6.1-block-comment-style">
./releasenotes.xml:          <a href="http://checkstyle.sourceforge.net/reports/google-java-style.html">Google style</a>
./config_naming.xml:         <a href="http://checkstyle.sourceforge.net/reports/google-java-style.html#s5.3-camel-case">
```

**Comment by romani** [comment]
manually squashed , rebased, merged.

**Pull request closed without merging**